unity/PitayaExample/Packages/manifest.json

```diff
@@ -1,15 +1,21 @@
 {
   "dependencies": {
-    "com.unity.ide.rider": "1.1.4",
-    "com.unity.ide.vscode": "1.2.1",
-    "com.unity.test-framework": "1.1.16",
+    "com.unity.ide.rider": "1.2.1",
+    "com.unity.ide.visualstudio": "2.0.12",
+    "com.unity.ide.vscode": "1.2.4",
+    "com.unity.test-framework": "1.1.29",
+    "com.unity.toolchain.macos-x86_64-linux-x86_64": "1.0.0",
```

Comment:
Won't this break on windows? LibPitaya should run on Windows, so this could be a problem. We could maybe add a todo here to investigate this in the future, I don't think windows users are using this test project, considering that even the tests were not running.

Reply:
This toolchain package is only required for building this example project using IL2CPP. The way it is it would likely break on windows, but I added the other two toolchain packages (windows and linux) so that build can happen properly. PTAL?

Reply:
LGTM